.github/workflows/backport.yml

```diff
@@ -10,9 +10,85 @@ permissions:
   id-token: write
 
 jobs:
+  find-related-issue:
+    runs-on: ubuntu-24.04
+    if: >
+      github.event.act ||
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+          && contains(join(github.event.pull_request.labels.*.name, ','), 'backport-')
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport-')
+        )
+      )
+    outputs:
+      url: ${{ steps.issue.outputs.url }}
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # 4
+        with:
+          fetch-depth: 0
+          sparse-checkout: |
+            .github/bin/js
+          sparse-checkout-cone-mode: false
+
+      - shell: bash
+        run: npm ci --prefix .github/bin/js/
+
+      - uses: actions/github-script@5c56fde4671bc2d3592fb0f2c5b5bab9ddae03b1 # 7
+        id: issue
+        env:
+          PR_REPO: ${{ github.event.pull_request.base.repo.full_name }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          PR_HEAD: ${{ github.event.pull_request.head.ref }}
+          PR_ASSIGNEE: ${{ github.event.pull_request.assignee && github.event.pull_request.assignee.login || '' }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { getDevelopmentIssueForPullRequest } = await import('${{ github.workspace }}/.github/bin/js/node_modules/@shopware-ag/gh-project-automation/dist/index.mjs')
+            const issue = await getDevelopmentIssueForPullRequest({github, core, context}, process.env.PR_REPO, parseInt(process.env.PR_NUMBER), process.env.PR_HEAD, process.env.PR_ASSIGNEE)
+
+            core.setOutput('id', issue?.id || null)
+            core.setOutput('title', issue?.title || null)
+            core.setOutput('number', issue?.number || null)
+            core.setOutput('url', issue?.url || null)
+
+  compute-targets:
+    runs-on: ubuntu-24.04
+    if: >
+      github.event.act ||
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+          && contains(join(github.event.pull_request.labels.*.name, ','), 'backport-')
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport-')
+        )
+      )
```

Comment:
Unfortunately, I had to repeat this condition multiple times, since GHA [still doesn't support YAML anchors](https://github.com/actions/runner/issues/1182)...